packages/botonic-core/src/handoff.ts

```diff
@@ -167,6 +183,8 @@ async function _humanHandOff(
   onFinish: string,
   agentEmail = '',
   agentId = '',
+  forceAssignIfNotAvailable: boolean,
```

Comment:
should be true by default? This is how it works now no?

Reply:
yes

Reply:
I've not set any default value here because if it's set to `true`, then the `force_assign_if_not_available` parameter will be sent always in all handoffs and since the default value in the backend is also `true`, I preffer not to send the parameter if it's not defined by the developer.

@MarcosCA that's why I added the condition of `if (forceAssignIfNotAvailable !== undefined)` (regarding your next comment) in order to add the parameter only if the developer has set a value to it.

Also, sending always this new parameter  in all handoff actions (even if it's not set by the developer) it may break tests done in some bots.

Reply:
Even if the default is true in the backend, I'd rather set the default here explicitly as well.
@asastre I don't know what tests this might break, but we can't make design choices based on some tests we have somewhere. We need to make the right choice looking forward and fix any tests if needed.

Reply:
@ericmarcos @MarcosCA @JianChen-IT As you've suggested, I've set `true` as the default value for `forceAssignIfNotAvailable` attribute.